### DISCUSSION TITLE
Is there some documentation about using YimMenu?

### DISCUSSION THREAD

**Discussion by NormalIQUser** [YimMenu/YimMenu#1152] [Q&A]
I'm trying to create a tab below self, but when imitating I get hundreds of errors, I was wondering if there might be some documentation about it, for adding new tabs

Regards ♥

**Comment by NormalIQUser** [comment]
The error I get after compiling is this in particular:

Error LNK2019	unresolved external symbol "public: static void __cdecl big::view::unlocker(void)" (?unlocker@view@big@@SAXXZ) unresolved referenced in the function "public: __cdecl big::gui_service::gui_service(void)" (??0gui_service@big@@QEAA@XZ)	

As I mentioned before, maybe I am skipping a step.

Explain what I did:

1.- in gui_service.cpp 

added in enum class tabs:

```
		RECOVERY,
		UNLOCKER,

```

added in class gui_service final:

```
		    {
		        tabs::RECOVERY,
		        {
		            "GUI_TAB_RECOVERY",
		            nullptr,
		            {
		                {tabs::UNLOCKER, {"GUI_TAB_UNLOCKER", view::unlocker}},
		            },
		        },
		    },
```


2.- in view.hpp:

added in public:

`		static void unlocker();`


3.- I created a folder called "recovery"

4.- in the folder I created a file called view_unlocker.cpp

5.- in view_unlocker.cpp i added this code; 

```
#include "views/view.hpp"

namespace big
{
	void view::unlocker()
	{
		ImGui::SetWindowSize({0.f, (float)*g_pointers->m_resolution_y}, ImGuiCond_Always);

		components::sub_title("UNLOCKER"_T);
		ImGui::Separator();
		});
	}
}
```

6.- After the compile I get the error :c 


hope this is helpful

**Comment by Seanghost117** [comment]
You have a few problems with the above code, the first thing that I believe is the problem is the file is not yet being used by the project, you can check this by navigating to the top panel of visual studio and seeing what project it's being used in which is most likely misc as it's not. Yo fix this just open the cmakelist file and just ctrl + s and let it regenerate and it will pick up the file and see there is a body for the function, you can also handle this in the cmake section in the top bar but for simplicity just go for the prior. Next check your transaction there for the subtitle, you want to make sure you are using the _T.data(), finally that windows shouldn't always be 0 x and fill the whole screens y, check that or allow readjustment to size.
